core-tests/src/test/java/org/mule/runtime/core/mule/enricher/MessageEnricherTestCase.java

```diff
@@ -344,7 +343,7 @@ public void enricherConservesSameEventInstanceNonBlockingTargetNonBlocking() thr
 
         nullReplyToHandler.latch.await(RECEIVE_TIMEOUT, TimeUnit.MILLISECONDS);
 
-        assertThat(sensingNullMessageProcessor.event.getMessage(), not(sameInstance(in.getMessage())));
+        assertThat(sensingNullMessageProcessor.event.getMessage(), sameInstance(in.getMessage()));
```

Comment:
Less meaningful, but no harm asserting that message isn't copied.  Can remove if you prefer..